src/labeler.ts

```diff
@@ -97,10 +100,21 @@ async function getLabelGlobs(
   client: ClientType,
   configurationPath: string
 ): Promise<Map<string, StringOrMatchConfig[]>> {
-  const configurationContent: string = await fetchContent(
-    client,
-    configurationPath
-  );
+  let configurationContent: string;
+
+  try {
+    configurationContent = (
+      await readFile(configurationPath, {
+        encoding: 'utf8'
+      })
+    ).toString();
+  } catch (error) {
+    core.info(
+      `${warningPrefix} configuration file (path: ${configurationPath}) not found locally (${error}), fetching via the api`
```

Comment:
After you pull the latest changes from the `main` branch, please move `core.info` to the `try` block and add a check to make sure if the configuration file exists. If it exists, let's read its content, if not, then print a message and try to fetch it via API.